src/angular/hq/src/app/clients/client-details/client-details-summary/client-details-summary.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable, of } from 'rxjs';
+import { switchMap, catchError, map } from 'rxjs/operators';
+import { HQService } from '../../../services/hq.service';
+import { GetClientRecordV1 } from '../../../models/clients/get-client-v1';
+import { APIError } from '../../../errors/apierror';
+import { CommonModule } from '@angular/common';
+import { ErrorDisplayComponent } from '../../../errors/error-display/error-display.component';
+
+
+@Component({
+  selector: 'hq-client-details-summary',
+  standalone: true,
+  imports: [CommonModule, ErrorDisplayComponent],
+  templateUrl: './client-details-summary.component.html',
+})
+export class ClientDetailsSummaryComponent implements OnInit {
+  clientId?: string;
+  client?: Observable<GetClientRecordV1 | null>;
```

Comment:
By convention we should try to suffix any properties on our components with a `$`, this should be `client$`. We also don't need to make this nullable if we define the value inside the constructor (preferred).